packages/data-stores/src/reader/mutations/use-site-unsubscribe-mutation.ts

```diff
@@ -5,6 +5,7 @@ import { SiteSubscriptionsPages, SubscriptionManagerSubscriptionsCount } from '.
 
 type UnsubscribeParams = {
 	blog_id: number | string;
+	url?: string;
 };
 
 type UnsubscribeResponse = {
```

Comment:
The response object of the `/read/following/mine/delete` endpoint has the following form:

```js
{ subscribed: boolean }
```

In other words, the `UnsubscribeResponse` type matches the original endpoint used for external users.

Reply:
I'll make the other 2 members optional.